bin/reth/Cargo.toml

```diff
@@ -57,6 +57,9 @@ reth-node-core.workspace = true
 reth-node-builder.workspace = true
 reth-node-events.workspace = true
 reth-consensus.workspace = true
+reth-evm-optimism = { workspace = true, optional = true, features = [
+    "optimism",
+] }
```

Comment:
I don't want this import here, instead this should be re-exported from the node-optimism crate